locations/spiders/asics_de.py

```diff
@@ -0,0 +1,22 @@
+import json
+
+import scrapy
+
+from locations.dict_parser import DictParser
+from locations.items import GeojsonPointItem
+
+
+class AsicsDeSpider(scrapy.Spider):
+
+    name = "asics_de"
+    item_attributes = {"brand": "asics", "brand_wikidata": "Q327247"}
+    allowed_domains = ["www.asics.com"]
+    start_urls = ["https://cdn.crobox.io/content/ujf067/stores.json"]
+
+    def parse(self, response):
+        data_json = json.loads(response.text)
+        for store in data_json:
+            item = GeojsonPointItem()
+            item = DictParser.parse(store)
```

Comment:
`DictParser.parse` creates the item for us.

```suggestion
            item = DictParser.parse(store)
```